qemu/tests/block_hotplug_with_max_map_count.py

```diff
@@ -0,0 +1,55 @@
+from avocado.utils import process
+from virttest import env_process
+
+
+def run(test, params, env):
+    """
+    Block hotplug with vm.max_map_count test
+    1) Boot the VM with a small max_map_count
+    2) Hotplug 15 block devices
+    3) Login and check the VM status
```

Comment:
Step 3 looks not accurate, actually, you start up the VM, login, do hotplugs, check the status.

Reply:
Updated !